music21/harmony.py

```diff
@@ -1853,7 +1857,8 @@ def _parseFigure(self):
             try:
                 justInts = int(justInts)
             except ValueError:
-                raise ValueError  # Not a properly formatted chord, ignore it
+                raise ValueError(f"Invalid chord abbreviation '{st}'; must be one of "
+                    f"{VALID_ABBRS} or specify all alterations")
```

Comment:
as above.

But I don't think that printing out VALID_ABBRS is the right thing to do.  it'll be far too long.  Maybe tell users:  "See music21.harmony.VALID_ABBRS for a list of all abbreviations."  ?

Reply:
Makes sense -- I removed VALID_ABBRS as the info is already in CHORD_TYPES anyway and changed the error message to reference CHORD_TYPES instead.